.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: test
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
not sure how picky it is, but you could try removing the colon, 
```
on: 
  workflow_dispatch
  pull_request
  push:
    ...
```

and see if that triggers the workflows?